src/test/java/redis/clients/jedis/csc/ClientSideCacheFunctionalityTest.java

```diff
@@ -545,4 +539,41 @@ public void testNullValue() throws InterruptedException {
     }
   }
 
+  public static class CustomCache extends TestCache {
+    public CustomCache(int maximumSize, EvictionPolicy evictionPolicy) {
+      super(maximumSize, evictionPolicy, DefaultCacheable.INSTANCE);
+    }
+  }
+
```

Comment:
We can add such constructor in TestCache and remove this.

Reply:
aim is to get more coverage on CacheFactory. 
using TestCache to covers [this](https://github.com/atakavci/jedis/blob/74a41857aa016ad769692fd9d4463f49796e6175/src/main/java/redis/clients/jedis/csc/CacheFactory.java#L31), 
`            Constructor ctorWithCacheable = findConstructorWithCacheable(config.getCacheClass());`
and CustomCache to cover [this](https://github.com/atakavci/jedis/blob/74a41857aa016ad769692fd9d4463f49796e6175/src/main/java/redis/clients/jedis/csc/CacheFactory.java#L38)
`            Constructor ctor = getConstructor(config.getCacheClass());`

Reply:
I've actually suggested add `TestCache(int maximumSize, EvictionPolicy evictionPolicy)` constructor; so it would cover that test.

Reply:
@sazzad16 i moved it into TestCache, and add controls around it to CacheFactory